tests/ssl-opt.sh

```diff
@@ -8655,11 +8655,11 @@ run_test    "TLS1.3: Not supported version check: tls1_2 and tls1_3" \
 requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_2
 requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL
 run_test    "TLS1.3: handshake dispatch test: tls1_3 only" \
-            "$P_SRV min_version=tls1_3 max_version=tls1_3" \
-            "$P_CLI min_version=tls1_3 max_version=tls1_3" \
+            "$P_SRV debug_level=2 min_version=tls1_3 max_version=tls1_3" \
+            "$P_CLI debug_level=2 min_version=tls1_3 max_version=tls1_3" \
             1 \
-            -s "SSL - The requested feature is not available" \
-            -c "SSL - The requested feature is not available"
+            -s "tls13 server state:" \
```

Comment:
Probably better to check the states we have been through.

Reply:
But the state number will be changed after adding new state or change the state position in `mbedtls_ssl_states`
And can we add debug function to convert state to string ?

Reply:
Python scripts has been added. Please check

Reply:
See https://github.com/ARMmbed/mbedtls/pull/4942#discussion_r715482188, I think having a state number is just fine.

Reply:
Has moved `debug helps` to #4981 . And add number for it